crowd-ui/src/features/translationValidation/components/footer/Footer.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import './footer.scss';
+import { KeyboardArrowRight } from '@mui/icons-material';
+import { FormattedMessage } from 'react-intl';
+import { useDispatch, useSelector } from 'react-redux';
+import { selectLoading, selectSentences } from '../../slice';
+import { postUserData } from '../../action';
+
+export default function Footer() {
+  enum FooterType {
+  skip = 3,
+  }
+  const sentence: any = useSelector<any, any>(selectSentences);
+  const isLoading: any = useSelector<any, any>(selectLoading);
+  const dispatch = useDispatch();
+
+  let sentencesId: any = null;
+  if ('elementItem' in sentence) {
+    sentencesId = 'id' in sentence.elementItem ? sentence.elementItem.id : null;
+  }
+  const getSendData = () => ({
+    sentencesId,
+    userSentenceSelect: FooterType.skip,
+  });
+
+  return (
+  // eslint-disable-next-line react/jsx-no-useless-fragment
```

Comment:
why ignore?

Reply:
because of  <> tag in "return" has a errore